.github/file-filters.yml

```diff
@@ -30,7 +30,7 @@ typecheckable_rules_changed: &typecheckable_rules_changed
 
 # Trigger to apply the 'Scope: Frontend' label to PRs
 frontend_all: &frontend_all
-  - added|modified: '**/*.[tj]{s,sx}'
+  - added|modified: '**/*.{ts,tsx,js,jsx,mjs}'
```

Comment:
I’ve included `.mjs` files here because they are also frontend related files, e.g. the eslint config files are in `.mjs` format.